packages/core/root/core-root-server-internal/src/events/kibana_started.ts

```diff
@@ -0,0 +1,173 @@
+                type: 'float',
+                _meta: {
+                  description:
+                    'Number of milliseconds the Node.js process has been running until the constructor was called',
+                },
+              },
+              end: {
+                type: 'float',
+                _meta: {
+                  description:
+                    'Number of milliseconds the Node.js process has been running until the constructor finished',
+                },
+              },
+            },
+          },
+          preboot: {
+            properties: {
+              start: {
+                type: 'float',
+                _meta: {
+                  description:
+                    'Number of milliseconds the Node.js process has been running until `preboot` was called',
+                },
+              },
+              end: {
+                type: 'float',
+                _meta: {
+                  description:
+                    'Number of milliseconds the Node.js process has been running until `preboot` finished',
+                },
+              },
+            },
+          },
+          setup: {
+            properties: {
+              start: {
+                type: 'float',
+                _meta: {
+                  description:
+                    'Number of milliseconds the Node.js process has been running until `setup` was called',
+                },
+              },
+              end: {
+                type: 'float',
+                _meta: {
+                  description:
+                    'Number of milliseconds the Node.js process has been running until `setup` finished',
+                },
+              },
+            },
+          },
+          start: {
+            properties: {
+              start: {
+                type: 'float',
+                _meta: {
+                  description:
+                    'Number of milliseconds the Node.js process has been running until `start` was called',
+                },
+              },
+              end: {
+                type: 'float',
+                _meta: {
+                  description:
+                    'Number of milliseconds the Node.js process has been running until `start` finished',
+                },
+              },
+            },
+          },
+          elasticsearch: {
+            properties: {
+              waitTime: {
+                type: 'float',
+                _meta: {
+                  description:
+                    'Number of milliseconds Kibana waited for Elasticsearch during the its start phase',
+                },
+              },
+            },
+          },
+          savedObjects: {
+            properties: {
+              migrationTime: {
+                type: 'float',
+                _meta: {
+                  description: 'Number of milliseconds it took to run the SO migration',
+                },
+              },
+            },
+          },
+        },
```

Comment:
Tricky to spot because of the new file, so those are the fields that were added in this PR.